ir/base.def

```diff
@@ -284,20 +295,20 @@ class Annotations {
         annotations.push_back(annot);
         return this; }
     Annotations add(Annotation annot) const { return clone()->add(annot); }
-    Annotations addAnnotation(cstring name, Expression expr) {
-        return add(new Annotation(name, Vector<Expression>(expr))); }
-    Annotations addAnnotation(cstring name, Expression expr) const {
-        return add(new Annotation(name, Vector<Expression>(expr))); }
+    Annotations addAnnotation(cstring name, Expression expr, bool structured) {
```

Comment:
You didn't want to use a default value of `false` for these methods to avoid having to add `false` in all the existing code?

Reply:
There are lots of overloads, and the preprocessor we wrote generates even more, so I wasn't sure which one will get used.